js/qmwp-search-correlations.js

```diff
@@ -134,6 +159,23 @@ quantimodoSearch.controller('QuantimodoSearchController', ['$scope', 'Quantimodo
             $scope.searchVariable = QuantimodoSearchConstants.predefinedVariable;
             $scope.showCorrelations($scope.searchVariable);
         }
+
+        function updateVotedCorrelations() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['updateVotedCorrelations' is defined but never used.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1175908137/issues/source?bid=2836188&fileBranchId=2920508#l163)